cowprotocol/cowswap
# Move external tokens to cow-react with SWR

# Summary

As per feedback, this PR updates the code for external token sources to;

* Live under cow react
* Use SWR instead of Jotai Loadable

It also includes the relevant image logic; as the previous PR (#2177) would not have been feasible anymore with said changes.

![Screenshot 2023-03-24 at 21 18 38](https://user-images.githubusercontent.com/5172699/227630754-9c09e746-2e33-481a-98fa-f35cdc506a05.png)

  # To Test

1. Open swap
2. Click on a currency to open search
3. Type a currency such as FORT
4. This should show a list with external sources added
5. This should show images for currencies such as FORTUNE

  # Background

https://cowservices.slack.com/archives/C0361CDG8GP/p1679678257447359
